src/openloco/input/mouse_input.cpp

```diff
@@ -857,6 +857,16 @@ namespace openloco::input
         }
     }
 
+    // 0x004C87B5
+    static void sub_4C87B5()
```

Comment:
Close and reset tooltip.
This ensures that a tooltip of the same type as was just closed can be reopened again. This is unlike the hover for over 8000 ticks which prevents the same tooltip from opening again on the current hover.

Reply:
All right. Do we have the same (similar) function in openrct2?

Reply:
As we discussed on discord the name will be changed to tooltip::closeAndReset()